src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -107,21 +102,15 @@ const UserProfile = ({
     );
   };
   return (
-    <>
-      <View style={styles.container}>
-        {userProfile()}
-        <ToggleSwitch
-          value={isAnonymous}
-          onValueChange={() => setTypeUser(!isAnonymous)}
-          labelLeft="Anonymity"
-          backgroundActive={colors.lightgrey}
-          backgroundInactive={colors.lightgrey}
-          circleInActiveColor={colors.blue1}
-          inactiveTextColor={colors.blue1}
-          styleLabelLeft={styles.switch}
-        />
-      </View>
-    </>
+    <View style={styles.container}>
+      {userProfile()}
+      <ToggleSwitchAnon
+        value={isAnonymous}
+        onValueChange={() => setTypeUser(!isAnonymous)}
+        labelLeft="Anonymity"
+        labelColor={COLORS.gray8}
+      />
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitch` component has been replaced with `ToggleSwitchAnon`. The properties `backgroundActive`, `backgroundInactive`, `circleInActiveColor`, `inactiveTextColor`, and `styleLabelLeft` have been removed, and `labelColor` has been added. Ensure that these changes do not affect the functionality of the switch and that the appearance is still as expected.
```diff
-        <ToggleSwitch
-          value={isAnonymous}
-          onValueChange={() => setTypeUser(!isAnonymous)}
-          labelLeft="Anonymity"
-          backgroundActive={colors.lightgrey}
-          backgroundInactive={colors.lightgrey}
-          circleInActiveColor={colors.blue1}
-          inactiveTextColor={colors.blue1}
-          styleLabelLeft={styles.switch}
-        />
+        <ToggleSwitchAnon
+          value={isAnonymous}
+          onValueChange={() => setTypeUser(!isAnonymous)}
+          labelLeft="Anonymity"
+          labelColor={COLORS.gray8}
+        />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->